reference/readonly-sdks/java/OpenAPI.json

```diff
@@ -17190,6 +17195,38 @@
           }
         }
       },
+      "ParameterFiltersListMemberTransactions": {
+        "title": "ParameterFiltersListMemberTransactions",
+        "type": "object",
+        "properties": {
+          "created_at": {
+            "title": "ParameterFiltersListMemberTransactionsCreatedAt",
+            "type": "object",
+            "description": "Filters the results by the date and time when the transaction occurred.",
+            "properties": {
+              "conditions": {
```

Comment:
dlaczego nie chcemy tu użyć FilterConditionsDateTime ?

Reply:
Fixed in: https://github.com/voucherifyio/voucherify-openapi/pull/832/commits/461b6b97d4943b47eb89998e1356d40c5edbaf30